docs/pages/database-access/guides/azure-sql-server-ad.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Database Access with Azure SQL Server with Azure Active Directory authentication (Preview)
+description: How to configure Teleport Database Access with Azure SQL Server using Azure Active Directory authentication.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="11.0"
+>
+  Database Access for Azure SQL Server with Azure Active Directory authentication
+  is available starting from Teleport `11.0`.
+</Details>
+
+<Admonition type="warning" title="Preview">
+  Database Access for Azure SQL Server with Azure Active Directory authentication is
+  currently in Preview mode.
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up access to Azure SQL Server using Azure Active Directory authentication.
+- Connect to Azure SQL Server through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Azure SQL Server Azure Active Directory Self-Hosted](../../../img/database-access/guides/sqlserver/sql-aad.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Azure SQL Server Azure Active Directory Cloud](../../../img/database-access/guides/sqlserver/cloud-sql-aad.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- SQL Server running on Azure.
+- The Teleport Database Service running on an Azure virtual instance.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+## Step 1/7. Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/7. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/7. Enable the SQL Server Azure Active Directory integration
+
+If you have it enabled, you can go to the next step.
+
+* Go to the [Azure Portal](https://portal.azure.com/), select **Database
```

Comment:
I think we can remove the bullets from this list as well and just use plain paragraphs.  For me, since these list items are complete sentences, there's not really a reason to make this a list.